draft-ietf-taps-interface.md

```diff
@@ -2388,6 +2408,33 @@ The Capacity Profile for a selected protocol stack may be modified on a
 per-Message basis using the Transmission Profile Message Property; see
 {{send-profile}}.
 
+### Ojective of Using Multiple Paths {#multipath-objective}
+
+Name:
+: multipath-objective
+
+Type:
+: Enumeration
+
+Default:
+: Handover
+
+This property specifies the local objective of transferring data across multiple paths between the same end hosts if Parallel Use of Multiple Paths not set to Disabled (see {{multipath-mode}}). Possible values are:
+
+None:
+: Do not use other paths as long as the initial path is available, but allow the peer endpoint to make use it to achieve its objective.
+
+Handover:
+: The connection should only attempt to migrate between different paths when the original path is lost or becomes unreliable.
```

Comment:
Rather than unreliable, let's say "unusable". Unreliable could just mean it has some loss—many links aren't reliable, but are good enough to use.

Reply:
This depends on whether to include "None" or not – if we drop "None", we should change it to unusable, but otherwise, I see more freedom to declare a path "bad enough to hand over to something else".

Reply:
+1 to unusable or another term. unreliable doesn't seem right